tests/containers/host_configuration.pm

```diff
@@ -21,11 +21,14 @@ use version_utils 'check_os_release';
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
+    my $interface = script_output qq(ip r | awk '{print \$5}' | head -1);
     if (check_os_release('suse', 'PRETTY_NAME')) {
+        validate_script_output "ip a s $interface", sub { m/((\d{1,3}\.){3}\d{1,3}\/\d{1,2})/ };
```

Comment:
```suggestion
        validate_script_output "ip a s '$interface'", sub { m/((\d{1,3}\.){3}\d{1,3}\/\d{1,2})/ };
```

so that it fails if `$interface` is somehow empty

Reply:
yes